docs/iterations/iter5/plan-2026-08-03-cleanup-campaign.md

```
@@ -0,0 +1,101 @@
+### D. 决策依赖(先 grill / 先 eval)
+| # | 内容 | 依赖 |
+|---|---|---|
+| D1 | vision 供应商对比 eval(MiMo vs gemini-flash 认番剧命中率) | 决策 2 |
+| D2 | 1-10-50 规约:补 worker/catalog lint 强制 或 修订规约 | 决策 6 |
+| D3 | logfire _internal scrubbing 耦合:启动断言 + 版本区间锁 | Py #14 |
+
+## S0 符合性核验结果(2026-08-03 第四路报告)
+
+**可安全关闭**:#231(S0.2)、#263(S0.10)、#248(S0.7)。已关的 #228、#244 复核无误。
+**可关但注记取代**:#234(tag-deploy→push-main+手动 dispatch;NextHandler 层随 #537 消亡)、
+#246(Supabase→Neon Auth = SD-31 决策;两条 browser AC 实际只有 unit 覆盖)。
```

markdownlint-cli2 (0.23.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)